var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -2,18 +2,19 @@
 # Spack Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
-
-from spack import *
-
 import re
 import os
 
+import spack.package
```

Comment:
I would be nice if this wasn't something that needed importing. Could we import it automatically like we do `spack.pkgkit.*`? We should talk about removing `from spack import *` someday, or adding `from spack.pkgkit import *`.